scanner/src/main/kotlin/scanners/Askalono.kt

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2017-2018 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.scanner.scanners
+
+import ch.frankel.slf4k.*
+
+import com.here.ort.scanner.Scanner
+import com.here.ort.utils.ProcessCapture
+import com.here.ort.utils.log
+import com.here.ort.utils.yamlMapper
+
+import java.io.File
+
+/**
+ * Global variable that gets set by a command line parameter parsed in the main entry points of the modules.
+ */
+var askalonoExecutablePath = ""
+
+data class AskalonoResult(
+        val filePath: String,
+        val output: String,
+        val error: String
+)
+
+object Askalono : Scanner() {
+    override val resultFileExtension = "yml"
+
+    private val possibleLicenseFiles = listOf("license", "copying", "readme")
```

Comment:
Was there a specific reason to run the tool only for those files? For example scancode we run for all files, because also source code often contains license information.